u2flib_server/u2f_v2.py

```diff
@@ -134,11 +141,15 @@ def __str__(self):
         return self.data.encode('hex')
 
     def verify_signature(self, pubkey):
-        data = self.app_param + self.user_presence + self.counter + \
-            self.chal_param
+        data = (self.app_param + self.user_presence + self.counter +
+                self.chal_param)
         digest = sha_256(data)
+
         pub_key = pub_key_from_der(pubkey)
-        if not pub_key.verify_dsa_asn1(digest, self.signature) == 1:
+
+        try:
+            verify_ecdsa_signature(digest, pub_key, self.signature)
+        except InvalidSignature:
```

Comment:
Oh yea, that doesn't seem to make a ton of sense. I've eliminated that and now it just straight up calls `verify_ecdsa_signature()`